fenix/app/src/androidTest/java/org/mozilla/fenix/ui/SettingsSearchTest.kt

```diff
@@ -419,7 +419,6 @@ class SettingsSearchTest {
     // Test running on beta/release builds in CI:
     // caution when making changes to it, so they don't block the builds
     // Goes through the settings and changes the search suggestion toggle, then verifies it changes.
-    @Ignore("Failing, see: https://github.com/mozilla-mobile/fenix/issues/23817")
     @SmokeTest
     @Test
     fun verifyShowSearchSuggestionsToggleTest() {
```

Comment:
what changed that makes this test stable now? was the initial failure that let to disabling it a bug that was fixed?

Reply:
There were 2 reasons because of which this test was disabled.

The awesome bar, was the first component that was refactored to compose, as you would imagine, the related UI tests were unstable because of this.

Afterwards, we've noticed that sometimes the suggestions don't get displayed on cold run there's a bug for this [1813587](https://bugzilla.mozilla.org/show_bug.cgi?id=1813587) that still needs to be addressed by the development team.

Based on the fact that there were no obvious problems related to our suggestions or other functionalities that involved the awesome bar I've performed a status check and the UI test worked properly, as mentioned above successfully ran 100x on Firebase.

Reply:
cool, sounds good to me.